views/place-form.hbs

```diff
@@ -1,211 +1,219 @@
-    </div>
-    <div>
-      <label for="disabled-toilets">{{localText.disabledToilet}}</label>
-      <input id="disabled-toilets" type="checkbox" name="accessibility" value="Disabled toilets" {{checkOptionsValue place.accessibilityOptions "Disabled parking"}}>
-    </div>
-  </fieldset>
-
-  <fieldset>
-    <legend>{{localText.placeCategories}}</legend>
-    <div>
-      <label for="cafe">{{localText.cafe}}</label>
-      <input id="cafe" type="checkbox" name="categories" value="cafe" {{checkOptionsValue place.categories "cafe"}}>
-    </div>
-    <div>
-      <label for="music-venue">{{localText.musicVenue}}</label>
-      <input id="music-venue" type="checkbox" name="categories" value="music venue" {{checkOptionsValue place.categories "music venue"}}>
-    </div>
-    <div>
-      <label for="religios-site">{{localText.religiousSite}}</label>
-      <input id="religios-site" type="checkbox" name="categories" value="religious site" {{checkOptionsValue place.categories "religious site"}}>
-    </div>
-    <div>
-      <label for="restaurant">{{localText.restaurant}}</label>
-      <input id="restaurant" type="checkbox" name="categories" value="food and drink" {{checkOptionsValue place.categories "restaurant"}}>
-    </div>
-    <div>
-      <label for="education">{{localText.education}}</label>
-      <input id="education" type="checkbox" name="categories" value="education" {{checkOptionsValue place.categories "education"}}>
-    </div>
-    <div>
-      <label for="retail">{{localText.retail}}</label>
-      <input id="retail" type="checkbox" name="categories" value="retail" {{checkOptionsValue place.categories "retail"}}>
-    </div>
-    <div>
-      <label for="sport">{{localText.sport}}</label>
-      <input id="sport" type="checkbox" name="categories" value="sport" {{checkOptionsValue place.categories "sport"}}>
-    </div>
-    <div>
-      <label for="municipal">{{localText.municipal}}</label>
-      <input id="municipal" type="checkbox" name="categories" value="municipal" {{checkOptionsValue place.categories "municipal"}}>
-    </div>
-    <div>
-      <label for="healthcare">{{localText.healthcare}}</label>
-      <input id="healthcare" type="checkbox" name="categories" value="healthcare" {{checkOptionsValue place.categories "healthcare"}}>
-    </div>
-  </fieldset>
-
-  <input type="submit" value="{{localText.formSubmit}}">
+{{>header}}
+{{>options}}
+
+<div class="tc w-100 pt5 mt2">
+  <span class="dark-text">{{localText.formInfo}}</span>
+</div>
+{{#if edit}}
+  <form class="" action="/{{lang}}/edit-place/{{place._id}}" method="post">
+    <input type="hidden" name="_method" value="put">
+{{else}}
+  <form class="" action="/{{lang}}/add-place" method="post">
+    <input type="hidden" name="_method" value="post">
+{{/if}}
+<table class="f6 w-100 mw8 center bb b--turquoise bw2">
+  <tbody>
+    <tr>
+      <td class="w-20 dark-text f4 b pa3 tr">*{{localText.name}}</td>
+      <td class="w-100">
```

Comment:
Ok so you're going to find this annoying, but looking at it now having all the inputs have `w-100` looks wierd

I think change all of the `td`'s that have `w-100` to `w-70` and also all of the `input`s that have `w-100` to `w-70` as well

Reply:
![screen shot 2017-10-28 at 11 38 30](https://user-images.githubusercontent.com/23310908/32132870-afe94854-bbd4-11e7-84fd-49f05cd989d5.png)


vs

![screen shot 2017-10-28 at 11 38 51](https://user-images.githubusercontent.com/23310908/32132866-a38bd932-bbd4-11e7-9a3c-1d6c396a84d0.png)